### PR TITLE
Create Environment Variables Panel in new lambdas view

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:
- Introduce Environment Variables panel in new lambda view

**Related issue(s)**
https://github.com/kyma-project/console/issues/1689

### CODE REVIEW

File: components/generic-documentation/src/renderers/helpers/ApiSelector.tsx [hunk 1]
```diff
@@ -51,11 +51,9 @@ const ApiSelector: React.FunctionComponent<{
 
   const maxApiNameLength = filteredSources.length
     ? getApiNameLength(
-        filteredSources.reduce((prev, current) => {
-          return getApiNameLength(current) > getApiNameLength(prev)
-            ? current
-            : prev;
-        }),
+        filteredSources.reduce((prev, current) =>
```
Comment (magicmatatjahu) [inline components/generic-documentation/src/renderers/helpers/ApiSelector.tsx:54]: It is formatted by prettier.

File: core-ui/src/components/Lambdas/constants.js [hunk 2]
```diff
@@ -175,6 +175,69 @@ export const CODE_AND_DEPENDENCIES_PANEL = {
...
+  },
+  VARIABLE_TYPE: {
+    CUSTOM: {
+      TEXT: 'Custom',
+      TOOLTIP_MESSAGE: 'The Variable is provided by User.',
+    },
+    BINDING_USAGE: {
+      TEXT: 'Service Binding',
+      TOOLTIP_MESSAGE:
+        'The Variable is injected by Service Binding from "{serviceInstanceName}" Service Instance.',
```
[the remark this answers is not in the document]
Comment (kazydek) [reply core-ui/src/components/Lambdas/constants.js:236]: And these from the existing file (thx a million!):

```

1. all Lambdas to Functions

2. These (+ shouldn't there be full stops after all errors as well?):

-   RESOURCES_NOT_FOUND: "This namespace doesn't have any Lambdas yet.", -> "This Namespace doesn't have any Functions yet.",

-   LAMBDA: {
    ERROR_MESSAGE: `Error while fetching "{lambdaName}" Lambda: {error}`, -> `"{lambdaName}" couldn't be fetched due to this error: {error}`

-  LAMBDAS: {
    ERROR_MESSAGE: `Error while fetching Lambdas in "{namespace}" Namespace: {error}`, -> `Functions in "{namespace}" couldn't be fetched due to this error: {error}`

-   EVENT_ACTIVATIONS: {
    ERROR_MESSAGE: `Error while fetching available Events in "{namespace}" namespace: {error}`, -> `Events in "{namespace}" couldn't be fetched due to this error: {error}` 

-   EVENT_TRIGGERS: {
      ERROR_MESSAGE: `Error while fetching Event Triggers for "{lambdaName}" Lambda: {error}`, -> `Event Triggers couldn't be fetched for "{lambdaName}" due to this error: {error}`

-  SERVICE_BINDING_USAGES: {
      ERROR_MESSAGE: `Error while fetching injected Service Binding Usages to "{lambdaName}" Lambda: {error}` -> `Injected Service Binding Usages couldn't be fetched for "{lambdaName}" due to this error: {error}`

-  SERVICE_INSTANCES: {
      ERROR_MESSAGE: `Error while fetching Service Instances: {error}`, -> `Service Instances couldn't be fetched due to this error: {error}`



- CREATE_LAMBDA: {
    SUCCESS_MESSAGE: `"{lambdaName}" Lambda is successfully created`, -> `"{lambdaName}" was successfully created`
    ERROR_MESSAGE: `Error while creating "{lambdaName}" Lambda: {error}`, -> `"{lambdaName}" couldn't be created due to this error: {error}`

- UPDATE_LAMBDA: {
    GENERAL_CONFIGURATION: {
      SUCCESS_MESSAGE: `General Configuration for "{lambdaName}" Lambda are successfully updated`,  -> `General configuration for "{lambdaName}" was successfully updated`,
      ERROR_MESSAGE: `Error while updating General Configuration for "{lambdaName}" Lambda: {error}`, -> `General configuration for "{lambdaName}" couldn't be updated due to this error: {error}`

-   CODE_AND_DEPENDENCIES: {
      SUCCESS_MESSAGE: `Code and Dependencies for "{lambdaName}" Lambda are successfully updated`, -> `Code and dependencies for "{lambdaName}" were successfully updated`,
      ERROR_MESSAGE: `Error while updating Code and Dependencies for "{lambdaName}" Lambda: {error}`, -> `Code and dependencies for "{lambdaName}" couldn't be updated due to this error: {error}`

-   RESOURCES_AND_REPLICAS: {
      SUCCESS_MESSAGE: `Resources and Replicas for "{lambdaName}" Lambda are successfully updated`, -> `Resources and replicas for "{lambdaName}" were successfully updated`
      ERROR_MESSAGE: `Error while updating Resources and Replicas for "{lambdaName}" Lambda: {error}`, -> `Resources and replicas for "{lambdaName}" couldn't be updated due to this error: {error}`

-     VARIABLES: {
      SUCCESS_MESSAGE: `Environment Variables for "{lambdaName}" Lambda are successfully updated`, -> `Environment variables for "{lambdaName}" were successfully updated`,
      ERROR_MESSAGE: `Error while updating Environment Variables for "{lambdaName}" Lambda: {error}`, -> `Environment variables for "{lambdaName}" couldn't be updated due to this error: {error}`

-   DELETE_LAMBDA: {
    SUCCESS_MESSAGE: `"{lambdaName}" Lambda is successfully removed`, -> `"{lambdaName}" was successfully deleted`,
    ERROR_MESSAGE: `Error while deleting "{lambdaName}" Lambda: {error}`, -> `"{lambdaName}" couldn't be deleted due to this error: {error}`

-     CONFIRM_MODAL: {
      TITLE: `Remove {lambdaName} Lambda`, -> `Delete {lambdaName}`
      MESSAGE: `Are you sure you want to delete "{lambdaName}" Lambda and whole related to it resources?`, -> `Are you sure you want to delete "{lambdaName}" and all related resources?`


- ERROR_MESSAGE_SINGLE: `Error while creating an Event Trigger for "{lambdaName}" Lambda: {error}`, -> `Event Trigger for "{lambdaName}" couldn't be created due to this error: {error}`
  ERROR_MESSAGE_MANY: `Error while creating Event Triggers for "{lambdaName}" Lambda: {error}`, -> `Event Triggers for "{lambdaName}" couldn't be created due to this error: {error}`


-   DELETE_TRIGGER: {
    SUCCESS_MESSAGE: `Event Trigger was successfully removed`,
    ERROR_MESSAGE: `Error while deleting "{triggerName}" Event Trigger for "{lambdaName}" Lambda: {error}`, -> `The "{triggerName}" Event Trigger for the "{lambdaName}" Function couldn't be deleted due to this error: {error}`
    CONFIRM_MODAL: {
      TITLE: `Remove Event Trigger`,
      MESSAGE: `Are you sure you want to delete "{triggerName}" Event Trigger for "{lambdaName}" Lambda?`, ->  `Are you sure you want to delete the "{triggerName}" Event Trigger for the "{lambdaName}" Function?`,

-       NOT_ENTRIES_POPUP_MESSAGE:
        'No Events available to connect in current namespace.', ->   'No Events available to connect in this Namespace.',

```

[the remark this answers is not in the document]
Comment (magicmatatjahu) [reply core-ui/src/components/Lambdas/constants.js]: <3

[the remark this answers is not in the document]
Comment (magicmatatjahu) [reply core-ui/src/components/Lambdas/constants.js]: ❤️

File: shared/components/ListActions/ListActions.js [hunk 3]
```diff
@@ -15,6 +15,7 @@ const StandaloneAction = ({ action, entry }) => {
       option="light"
       glyph={icon}
       aria-label={action.name}
+      typeAttr="button"
```
Comment (magicmatatjahu) [inline shared/components/ListActions/ListActions.js:18]: If GenericList is in ModalWithForm, then buttons has `submit` type, so I changed to normal `button` type

**Pull request merged**